treeherder/model/fixtures/repository_group.json

```diff
@@ -4,23 +4,23 @@
     "model": "model.repositorygroup",
     "fields": {
         "name": "development",
-        "description": ""
+        "description": "Collection of repositories where code initially lands in the development process"
```

Comment:
I think some of these could be made a bit more concise, but happy to go with whatever you choose :-)